### PR TITLE
Follow up

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
Please provide comments

### CODE REVIEW

File: gcn/notices/core/FollowUp.schema.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/core/FollowUp.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "FollowUp",
+  "description": "Schema specific to follow-up cases",
```
Comment (jracusin) [inline gcn/notices/core/FollowUp.schema.json:5]:
```suggestion
  "description": "Schema specific to follow-up observations or searches",
```

**Comment by Vidushi-GitHub** [comment]
@Tohuvavohu, @shb46, @blaufuss and @joshuarwood, this follow-up core schema is new addition. Please feel free to implement it in your schema, and we welcome any suggestions. @jracusin @dakota002

**Comment by joshuarwood** [comment]
Ok, thanks. The earliest I can implement this for GBM is during the first O4 break. Our LVK collaborators needed the GBM Targeted Search alert format frozen by yesterday. They're struggling to get their consumers ready in advance of the official O4 start next week.

For now, I have a fork with the Targeted Search format based on the prior commits to the main GCN schema:
https://github.com/joshuarwood/gcn-schema/blob/main/gcn/notices/fermi/gbm/TargetedSearch.example.json
https://github.com/joshuarwood/gcn-schema/blob/main/gcn/notices/fermi/gbm/TargetedSearch.schema.json

We're sending these alerts through a private GCN stream so it should be Ok if they don't fully match the official schema just yet. I'll do my best to match it to the official schema once it's out of our critical path for receiving real time alerts.

**Comment by Tohuvavohu** [comment]
Right @lpsinger should be aware of this, and we have already discussed the relevant timelines with Judy, etc.
If Cody+Brnadon needed the alert format frozen yesterday, I likewise will not be able to change it again.

**Pull request merged**